docs/sdk/dev/value-resolvers.md

```diff
@@ -3,25 +3,25 @@ title: Value resolvers
 description: The Spryker SDK has several value resolvers described in this document.
 template: concept-topic-template
 ---
+# Spryker SDK value resolvers
```

Comment:
as you already have the title in the first line, using one more H1 title is redundant